EngineLayer/CompactToProteinMatchEngine/SequencesToActualProteinPeptidesEngine.cs

```diff
@@ -1,4 +1,6 @@
-﻿using Proteomics;
+﻿using MassSpectrometry;
+using Proteomics;
```

Comment:
These are different namespaces, kind of like how `using System;` and `using System.IO;` are different. I made this distinction because there are two `Peptide` objects in `Proteomics` with the addition of the digestion code.